### ISSUE TITLE
[Bug][A11Y][Phone Number] Prefix is not read by screen readers

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#498]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

.

### OS Version

Android and iOS

### App version

0.6.0 (509)

### Describe the issue

Phone prefix is not read by screen-readers

<img width="455" height="716" alt="" src="https://github.com/user-attachments/assets/f864bccd-8c69-446d-8c20-5306af1d82a9" />

### Expected Behavior

Prefix should be read like on Text Input component

**Comment by B3nz01d** [comment]
@AhmedAmineZr on ios 591, it reads the prefix and then a "vertical line" after the prefix

**Comment by AhmedAmineZr** [comment]
> [@AhmedAmineZr](https://github.com/AhmedAmineZr) on ios 591, it reads the prefix and then a "vertical line" after the prefix

I tested into android and iOS and I don't have vertical line readed by voice over

**Comment by AhmedAmineZr** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (alpha) (592)
Version: 0.7.0
Build type: Alpha
Build details (GitHub): 326
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (Accessibility)

**Comment by AhmedAmineZr** [comment]
📣 New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: Design Toolbox Flutter (592)
Version: 0.7.0
Build type: Alpha
Build details (GitHub): 326
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (Accessibility)

**Comment by pya35** [comment]
@AhmedAmineZr Clos

**Comment by B3nz01d** [comment]
Finished during sprint # 25
